messages/main/en.json

```
@@ -9,6 +9,7 @@
   "menu-file": "File",
   "menu-change-language": "Change language...",
   "menu-change-language-title": "Enter the language (e.g., 'en' or 'es')",
+  "menu-config-complete": "Successfully Imported Config: ",
```

Should be sentence-cased:

```suggestion
  "menu-config-complete": "Successfully imported config: ",
```